### PR TITLE
Add SubMasterbarNav to theme-showcase view for not logged users

### PR DESCRIPTION
**Pull request by ice9js**
This pull-request adds the ```SubMasterbarNav``` to ```theme-showcase``` section for not logged users.  
The links are a subset of `subject:` filter and should redirect to `/themes/{subject}`.

themes_submasterbarnav

Testing:

- Go to `/themes` - `SubMasterbarNav` component should appear under the `Masterbar`
- Check if the links in `SubMasterbarNav` redirect to `/themes/{filter}`
- `SubMasterbarNav` isn't fixed and should move out of the viewport once your start scrolling
- Check if `SubMasterbarNav` doesn't interfere with the search bar on different viewport widths

### CODE REVIEW

**Comment by ice9js** [comment]
I adjusted `SubMasterbarNav` a bit so that it always renders as expanded on the server, and collapses once it's rendered on the client if necessary. That's also when the 'expand' button would be added.

File: client/components/sub-masterbar-nav/navbar.jsx [hunk 1]
```diff
@@ -62,7 +63,9 @@ export default class Navbar extends Component {
 				</div>
 				{ this.state.foldable && (
 					<div className="sub-masterbar-nav__switch">
-						<Gridicon icon="ellipsis" className={ ellipsisClass } onClick={ this.toggleList } />
+						<Gridicon
+							icon="ellipsis"
+							className={ ellipsisClass } onClick={ this.toggleList } />
```
Comment (budzanowski) [inline client/components/sub-masterbar-nav/navbar.jsx]: onClick should be on next line if you have decided to break props into lines.

File: client/components/sub-masterbar-nav/navbar.jsx [hunk 2]
```diff
@@ -82,16 +85,20 @@ export default class Navbar extends Component {
...
 
 	toggleList = () => {
-		this.setState( ( state ) => ( {
-			collapsed: ! state.collapsed
+		this.setListCollapsed( ! this.state.collapsed );
+	}
+
+	setListCollapsed( collapsed ) {
+		this.setState( () => ( {
+			collapsed: collapsed
```
Comment (budzanowski) [inline client/components/sub-masterbar-nav/navbar.jsx]: Is there a benefit for having a callback here? Is this better than:
`this.setState( { collapsed } );`

**Comment by folletto** [comment]
Looks good!

Let's just avoid the initial jump:

cap-

**Comment by folletto** [comment]
Design wise 👍

File: client/components/sub-masterbar-nav/navbar.jsx (hunk 2 above)
Comment (ehg) [inline client/components/sub-masterbar-nav/navbar.jsx]: I think React 16 will default to this? To make it clear that `setState` is async. Not sure :)

**Comment by budzanowski** [comment]
No I have none :) Looks great!

**Comment by ockham** [comment]
Just saw it in production. Looks really great @ice9js!

**Pull request merged**